api/src/main/resources/beans_2_0.xsd

```diff
@@ -353,5 +354,14 @@
       </xs:choice>
     </xs:complexType>
   </xs:element>
+  <xs:element name="trim" type="xs:string" fixed="">
```

Comment:
I was just thinking if we could/should add some restrictive attributes here. Such as limiting min/max occurrences?